STM32/Temperature/Core/Src/Temperature.c

```diff
@@ -18,13 +18,15 @@
 // Local functions
 static void calibrateTypeInput(int noOfCalibrations, const CACalibration* calibrations);
 static void calibrateReadWrite(bool write);
-static void TempPrintHeader();
+static void printTempHeader();
+static void printTempStatus();
```

Comment:
I'd probably put `internalTemperature()` here too

Reply:
Why? :) It is not a function that is being used other than internally in the file.

Reply:
Typically I'd put prototypes for all the internal functions at the top of the file. It means you can put the public functions (which I'd argue are the most important) at the top of the .c and push the statics to the bottom. It also means you don't need to re-arrange the functions if you decide you want to call one from another.

BUT it is slightly more verbose. Its up to you, I wouldn't stop the PR over it :)

Reply:
At this point the private member functions have been placed at the top of the file. I'm not too concerned with whether the public are first or last, but I do think it is nice that the placement does not matter when using prototypes.